pkg/envoy/accesslog_server.go

```diff
@@ -20,16 +20,16 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
-	"syscall"
 	"time"
 
 	"github.com/cilium/cilium/pkg/flowdebug"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 	"github.com/cilium/cilium/pkg/proxy/logger"
 
-	"github.com/cilium/proxy/go/cilium/api"
+	cilium "github.com/cilium/proxy/go/cilium/api"
```

Comment:
Why is this line changing if there's no corresponding change in the rest of the file?

Reply:
I believe the problem was: `'goreturns' will add missing imports and remove unused imports.` 

Disabled.... vscode would make the change before saving and closing the file.. :man_facepalming:.

Reply:
Are there many vscode-specific configurations you need to apply for developing on Cilium? Just wondering if it's worth sharing the vscode configs in the Cilium repo or as documentation or even just as a gist somewhere so if someone searches "vscode cilium" they'll find the instructions?